docs/extras/modules/memory/agent_with_memory.ipynb

```diff
@@ -5,19 +5,19 @@
    "id": "fa6802ac",
    "metadata": {},
    "source": [
-    "# How to add Memory to an Agent\n",
+    "# How to add memory to an agent\n",
```

Comment:
Here it is about the specific classes with names: Memory and Agent, not generic memory and agent.